components/infobox/wikis/ageofempires/infobox_person_player_custom.lua

```diff
@@ -43,7 +43,7 @@ local CustomInjector = Class.new(Injector)
 
 local RATINGCONFIG = {
 	aoe2 = {
-		{text = 'RM [[Age of Empires II/Definitive Edition|DE]]', id = 'aoe2net_id', game = 'aoe2de'},
+		--{text = 'RM [[Age of Empires II/Definitive Edition|DE]]', id = 'aoe2net_id', game = 'aoe2de'},
```

Comment:
Remove rather than comment, it's still available in the commit history if/when it needs to be retrieved